model-api-gen-gradle-test/kotlin-generation/src/test/kotlin/org/modelix/modelql/typed/TypedModelQLTest.kt

```diff
@@ -371,4 +371,37 @@ abstract class TypedModelQLTest {
         }
         assertEquals("", name)
     }
+
+    @Test
+    fun `chained write operations are executed once`() = runTypedModelQLTest { client ->
```

Comment:
Nitpick: this test actually asserts more: it also validates that the correct write operations take place and result in the desired state.